apps/src/codebridge/components/ResizeBar.tsx

```diff
@@ -11,37 +10,63 @@ interface ResizeBarProps {
   separatorProps: SeparatorProps;
 }
 
-export const RESIZE_BAR_SIZE_PX = 13;
+export const RESIZE_BAR_SIZE_PX = 1;
 
+// A resize bar that can be dragged to resize two adjacent panels.
+// The bar is 1px wide, but when it is hovered/focused or being dragged, it
+// becomes 5px wide. The area that can be grabbed is also 5px wide.
```

Comment:
It might be worth elaborating that the visible bar element that is "[in flow](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flow_layout/In_flow_and_out_of_flow)" remains 1px even when dragged, but there is an additional "out of flow" overlay that is always 5px and is used for grabbing and also becomes visible when in hover/focus/drag state.

Reply:
updated, let me know if it seems clear enough!